runtime/autoload/health/nvim.vim

```diff
@@ -0,0 +1,410 @@
+" Script variables
+let s:bad_responses = ['unknown', 'unable to parse', 'unable to get pypi response', 'unable to get neovim executable']
```

Comment:
I haven't finished that part. I was hoping to remove all of the `unknown` calls from the script, and replace them with more verbose names. Then I would check if the response was in the responses, and if so, then it would error. Currently we just check if it's `unknown` in a lot of places.

So it's not needed yet, but I hope to use it soon.